src/process_memory.rs

```diff
@@ -138,6 +138,6 @@ mod tests {
     fn test_max_rss() {
         // See if it's a sort of sane value, between 1 and 700 mb
         assert!(super::max_rss() > 1_000);
-        assert!(super::max_rss() < 700_000);
+        assert!(super::max_rss() != 0);
```

Comment:
```suggestion
```

We already test if it's not 0 with the assertion above it. Let's remove the assertion entirely.

Reply:
true 😅